src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -60,7 +60,9 @@ public ListPanel(UI ui, IModel model, PanelControl parentPanelControl, int panel
      * @return true if the issue has changed, false otherwise
      */
     private boolean issueHasNewComments(TurboIssue issue, boolean hasMetadata) {
-        if (currentFilterExpression.getQualifierNames().contains(Qualifier.UPDATED) && hasMetadata) {
+        List<String> filterQualifierNames = currentFilterExpression.getQualifierNames();
+        if ((filterQualifierNames.contains(Qualifier.UPDATED) ||
```

Comment:
Okay. I guess you could add a comment explaining this, since you're purposefully omitting the self check.